go/vt/vtctl/vtctl.go

```diff
@@ -2324,6 +2327,15 @@ func commandVReplicationWorkflow(ctx context.Context, wr *wrangler.Wrangler, sub
 		return fmt.Errorf("workflow %s does not exist", ksWorkflow)
 	}
 
+	if len(vrwp.ShardSubset) > 0 {
+		if workflowType == wrangler.MoveTablesWorkflow && action != vReplicationWorkflowActionCreate && wf.IsPartialMigration() {
+			log.Infof("Subset of shards: %s have been specified for keyspace %s, workflow %s, for action %s",
+				vrwp.ShardSubset, target, workflowName, action)
+		} else {
+			return fmt.Errorf("The --shards option can only be specified for existing Partial MoveTables workflows")
```

Comment:
Nit, but error messages should not be capitalized (linter should warn about this).

Reply:
Fixed. The linter is no longer warning about this. Not sure if we removed it explicitly or it was inadvertent. Or CI static checks should fail, I guess, if we are checking for this.